docs/reference/commandline/network_create.md

```diff
@@ -122,6 +122,24 @@ docker network create -d overlay
 ```
 Be sure that your subnetworks do not overlap. If they do, the network create fails and Engine returns an error.
 
+# Specifying specific options for the bridge driver
+
+When creating a custom network, the default network driver (i.e. `bridge`) has additional options that can be passed.
+The following are those options and the equivalent docker flags used for docker0 bridge.
+
+* `com.docker.network.bridge.name` : Specify a bridge name to be used when creating the Linux bridge
+* `com.docker.network.bridge.enable_ip_masquerade' : Equivalent to docker daemon's `-ip-masq` flag
+* `com.docker.network.bridge.enable_icc' : Equivalent to docker daemon's `-icc` flag
+* `com.docker.network.bridge.host_binding_ipv4' : Equivalent to docker daemon's `--ip` flag
+* `com.docker.network.mtu' : Equivalent to docker daemon's `--mtu` flag
+* `com.docker.network.enable_ipv6' : Equivalent to docker daemon's `-ipv6` flag
```

Comment:
Looks like there are some "regular" quotes here, not back-tics. Wondering if we should use a table here, e.g.

```Markdown
| Option                                           | Equivalent                | Description
| ------------------------------------------------ | ------------------------- | ------------------------------------------------------- 
| `com.docker.network.bridge.name`                 | -                         | bridge name to be used when creating the Linux bridge  |
| `com.docker.network.bridge.enable_ip_masquerade` | `docker daemon --ip-masq` | Enable IP masquerading                                 |
| `com.docker.network.bridge.enable_icc`           | `docker daemon --icc`     | Enable addition of iptables rules                      |
| `com.docker.network.bridge.host_binding_ipv4`    | `docker daemon --ip`      | Default IP when binding container ports                |
| `com.docker.network.mtu`                         | `docker daemon --mtu`     | Set the containers network MTU                         |
| `com.docker.network.enable_ipv6`                 | `docker daemon --ipv6`    | Enable IPv6 networking                                 |

```

Not sure if we should keep the middle ('equivalent') column, but open to suggestions